specification/sources/chapters/extensions/oppo/oppo_controller_interaction.adoc

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2023 OPPO
+//
+//
+
+
+include::../meta/XR_OPPO_controller_interaction.adoc[]
+
+*Contributors*::
+    Haomiao Jiang, OPPO +
+    Buyi Xu, OPPO +
+    Yebao Cai, OPPO +
+
+*Overview*
+
+This extension defines a new interaction profile for the OPPO Controller, including but not limited to OPPO MR Glasses Controller.
+
+*OPPO Controller interaction profile*
+
+Interaction profile path:
+* pathname:/interaction_profiles/oppo/oppo_mr_controller
+
+Valid for user paths:
+
+* pathname:/user/hand/left
+* pathname:/user/hand/right
+
+This interaction profile represents the input sources and haptics on the OPPO Controller.
+
+Supported component paths:
+* On /user/hand/left only:
+** subpathname:/input/x/click
+** subpathname:/input/x/touch
+** subpathname:/input/y/click
+** subpathname:/input/y/touch
+** subpathname:/input/menu/click
+** subpathname:/input/heartrate_oppo/value
```

Comment:
You will want to define the meaning of this, mainly the units (beats per minute?). See the "New Identifiers" and "Input Path Descriptions" sections of https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XR_FB_touch_controller_pro